app/frontend/components/pages/delegations/delegatePage.tsx

```diff
@@ -46,58 +49,148 @@ const BigDelegatorOffer = (): h.JSX.Element => (
   </div>
 )
 
+// REFACTOR: (Untyped errors): move to types
+// is "hasTickerMapping" something specific or general?
```

Comment:
?

Reply:
Something specific, we implemented a "search by ticker" a couple of months ago. We receive pool info from backend but in some cases we don't receive the tickers. (not much we can do about that). In that case, if a user entered a ticker, we would like to show him a specific error that `search by ticker is temporarily disabled` rather than something general (and misleading) like `Invalid stakepool identifier)`